modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -28,13 +30,17 @@ dependencies {
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "2.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "2.0.0"
 #elseif (${liferayVersion.startsWith("7.1")})
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib", version: "3.2.2"
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib.util.freemarker.contributor", version: "2.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "3.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "3.0.2"
-	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "3.1.2"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "3.4.2"
 #elseif (${liferayVersion.startsWith("7.2")})
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib", version: "4.0.15"
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib.util.freemarker.contributor", version: "3.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "4.4.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "6.0.2"
-	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "4.0.8"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "4.1.0"
```

Comment:
Does this version match 7.2 GA1 or upcoming release?

Reply:
I matched it to 7.2 GA1 BOM. I thought it was originally matched to the GA1 BOM already, but maybe i'm wrong. Or they reuploaded a newer BOM